src/wallet/cc_wallet/cc_info.py

```diff
@@ -1,14 +1,22 @@
 from dataclasses import dataclass
 from typing import List, Optional, Tuple
 
+from src.types.program import Program
 from src.types.sized_bytes import bytes32
 from src.util.streamable import streamable, Streamable
+from src.wallet.cc_wallet import cc_wallet_puzzles
 from src.wallet.cc_wallet.ccparent import CCParent
 
 
 @dataclass(frozen=True)
 @streamable
 class CCInfo(Streamable):
-    my_core: Optional[str]  # core is stored as the disassembled string
     parent_info: List[Tuple[bytes32, Optional[CCParent]]]  # {coin.name(): CCParent}
-    my_colour_name: Optional[str]
+    my_colour_name: str
```

Comment:
This has not changed but I think we should rename this variable to just "colour", "my_colour_name" is easy to confuse with wallet name that is given by user.

Reply:
I agree, and standardized names should be used. I prefer the name `genesis_id` over `colour`.

Reply:
my_colour_name is the nickname for the colour that shows up in the UI, it is not an essential piece of information.
By default it gets set to the genesis_id but can be changed, i.e to "MattCoin". 
I think clarifying that it is a nickname of some kind is important.

Reply:
Probably should add `mod_hash` here.